### ISSUE TITLE
Binary 10.1.2 release

### ISSUE THREAD

**Issue by Halamix2** [astrelsky/Ghidra-Cpp-Class-Analyzer#49]
Hello, can I ask for 10.1.2 release?

**Comment by astrelsky** [comment]
> Hello, can I ask for 10.1.2 release?

Yes of course. I didn't realize I hadn't put one out. I will do so in about an hour.
